config/README.md

```diff
@@ -0,0 +1,24 @@
+# Graph SDK Configuration
+<p>
+All of different settings and configuration parameters are encapsulated inside this module.
+</p>
+
+#### Supported `Environments` and their related configuration
+  - `Mainnet` : which is the **production** related settings
+  - `Rococo` : which is the **staging** related settings
+  - `Dev` : which is for **local** development and testing
+
+### Graphs and Schemas
+Due to `DSNP` implementation over `Frequency` Chain, we are associating each a specific and well-defined Avro schema
```

Comment:
This is not clear especially this part
`we are associating each a specific and well-defined Avro schema`

Reply:
Yeah much better